integration/integration_test.go

```diff
@@ -2948,78 +2892,77 @@ func (s *IntSuite) TestExternalClient(c *check.C) {
 	}
 
 	for _, tt := range tests {
-		// Create a Teleport instance with auth, proxy, and node.
-		makeConfig := func() (*check.C, []string, []*InstanceSecrets, *service.Config) {
-			recConfig, err := types.NewSessionRecordingConfig(types.SessionRecordingConfigSpecV2{
-				Mode: tt.inRecordLocation,
-			})
-			c.Assert(err, check.IsNil)
+		t.Run("", func(t *testing.T) {
```

Comment:
Nit: Can you add test description?

Reply:
Yep, will do.

Fun thing I learned: `T.Run()` will supply an ordinal name for you if you don't give a custom one, so these tests show up as `.../#00`, `.../#01`, etc. Which is nice.

Reply:
Done